bucket_map/src/bucket.rs

```diff
@@ -96,6 +96,10 @@ pub struct Bucket<T: Copy + 'static> {
     pub data: Vec<BucketStorage<DataBucket>>,
     stats: Arc<BucketMapStats>,
 
+    /// # entries caller expects the map to need to contain.
+    /// Used as a hint for the next time we need to grow.
+    anticipated_size: AtomicU64,
```

Comment:
ah. good point. well... I was imagining it only required a read lock to set it. But it is called at startup where there should be no lock contention. And we have a mut self, so it could be just a regular old u64...